dvsa/cvs-svc-defects
# Tech / Implement `esbuild` to bundle and minify code

## Ticket title
- Implement esbuild to minify and bundle code, the will decreased the ZIP artefact from `15.4mb` to ~`1.1mb`

To test out:
- Run `ZIP_NAME=someName npm run package` - This should create a `someName.zip` which contains a `handler.js` file

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number
